lib/hardware_services/network_drivers/HilNetworkDriver.py

```diff
@@ -14,19 +14,74 @@
 from subprocess import call
 from subprocess import check_call
 
-from hardware_services.network_service import NetworkService
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "util"))
 
-# added for EC528 HIL-QUADS integration project
-hil_url = 'http://127.0.0.1:5000'
+import hilapi
 
 class HilNetworkDriver(NetworkService):
 
+    def __read_current_state(self, argv):
+        """
+        Reads actual current network of a node from the HIL database and returns name of this network
+        """
+        
+        pass
+
+    def __write_current_state(self, argv):
+        """
+        Writes/updates information about a new network of a node to the HIL database
+        """
+        
+        pass
+        
+    def __move_one_host(self, argv):
+        if len(argv) != 3:
+            sys.exit("Incorrect number of arguments. Should be host_to_move old_cloud new_cloud.")
+        host_to_move = argv[0]
+        old_cloud = argv[1]
+        new_cloud = argv[2]
+        node_nics = hilapi.show_node(host_to_move)['nics']
+        print("TESTING: Before move " + str(node_nics))
+        for nic_json in node_nics:
+            
+            time.sleep(1)
+            hilapi.node_detach_network(host_to_move,
+                                nic_json['label'],
+                                old_cloud)
+            time.sleep(1)
+
+            time.sleep(1)
+            hilapi.node_connect_network(host_to_move,
+                                nic_json['label'],
+                                new_cloud,
+                                'null')
+            time.sleep(1)
+
+        node_nics = hilapi.show_node(host_to_move)['nics']
+        print("TESTING: After move " + str(node_nics))    
```

Comment:
Perhaps put this under a debug conditional, like: `if _DEBUG > 0:` and then have a module global you can play with if necessary.